src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -25,16 +25,18 @@
                 <shared:commentsPanel commentsForStudentsTables="${data.commentsForStudentTable}" courseId="${data.courseId}" forRecordsPage="${true}" />
                 <br>
                 <c:forEach items="${data.sessionNames}" var="fsName" varStatus="fbIndex">
-                    <div class="well well-plain student_feedback"
+                    <div class="student_feedback panel panel-default"
```

Comment:
Yeah, I know, but 'student_feedback' already existed before this commit, so I was confused is it mistake overlooked on previous reviews or not